frontend/messages/en.json

```
@@ -1384,6 +1384,10 @@
 	"requirementsProgressionSemiColon": "Requirements progression:",
 	"noRequirementFound": "No requirement assessment found",
 	"noAuditForTheFramework": "There is no audit using this framework",
+	"multipleChoice": "Multiple choice",
+	"uniqueChoice": "Unique choice",
+	"text": "Text",
+	"date": "Date",
```

Biome (1.9.4)

[error] The key date was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)